bs/src/tablecloth.ml

```diff
@@ -432,28 +432,46 @@ module Tuple2 = struct
   
   let second ((_, b) : 'a * 'b) : 'b = b
 
+  let map ~(f : 'b -> 'y) ((a, b) : 'a * 'b) : 'a * 'y = (a, f b)
+
   let mapFirst ~(f : 'a -> 'x) ((a, b) : 'a * 'b) : 'x * 'b = (f a, b)
 
   let map_first = mapFirst
 
-  let mapSecond ~(f : 'b -> 'y) ((a, b) : 'a * 'b) : 'a * 'y = (a, f b)
+  let mapSecond = map
 
   let map_second = mapSecond
 
-  let mapBoth ~(f : 'a -> 'x) ~(g : 'b -> 'y) ((a, b) : 'a * 'b) : 'x * 'y = (f a, g b)
+  let mapEach ~(f : 'a -> 'x) ~(g : 'b -> 'y) ((a, b) : 'a * 'b) : 'x * 'y = (f a, g b)
```

Comment:
Using `~g` is clever! I wonder if it might be better as `~firstf` or `~ffirst` or `~f1`, etc?

Reply:
Or maybe `~first` and `~second` to match the accessor functions?

Reply:
My local functional guru says `~f` and `~g` and `~h` is a known idiom, so might be better to do that than invent a new scheme? I'm mostly open to anything though.

Reply:
Ok, lets leave it with `f`, `g` &`h` then.